ci/default.nix

```diff
@@ -75,6 +75,30 @@ let
           includes = [ "*" ];
           priority = 1;
         };
+
+        # TODO: Upstream this into treefmt eventually?
+        settings.formatter.markdown-code-runner = {
+          # Remove shell script once this is solved:
+          #   https://github.com/drupol/markdown-code-runner/issues/6
+          # Remove sed step once this is solved:
+          #   https://github.com/drupol/markdown-code-runner/issues/7
```

Comment:
should be fixed now: https://github.com/drupol/markdown-code-runner/releases/tag/0.2.0

Reply:
And in https://github.com/NixOS/nixpkgs/pull/427516